downloads/swan-lake-release-notes/swan-lake-preview5/RELEASE_NOTE.md

```diff
@@ -0,0 +1,255 @@
+    int age = check j1.age;
+}
+```
+
+##### Improved lang library functions
+
+Several changes and improvements have been introduced to the `ballerina/lang.value` lang library module.
+
+###### New `ensureType` function
+
+A new `ensureType` function has been introduced to convert any kind of a Ballerina value to a user-specified type.
+
+###### New `toBalString()` and `fromBalstring()` functions
+
+Two new functions namely `toBalString()` and `fromBalstring()` have been introduced to roundtrip between `anydata` and Ballerina expression-like `string`. The `toBalString()` function converts a Ballerina value to a `string` that looks like a Ballerina expression (expression style). 
+
+The `fromBalstring()` function parses and evaluates a `string` produced by `toBalString()` when applied to an `anydata` value. The details of the conversion performed in `toBalString()` are specified by the `ToString` abstract operation defined in the Ballerina Language Specification using the expression style.
+
+###### More `isolated` lang library functions
+
+These include lang library functions that accept an argument of a `function` type. With these functions, a parameter, which is of a `function` type may be annotated with the `@isolatedParam` annotation. 
+
+```ballerina
+public isolated function forEach(Type[] arr, @isolatedParam function(Type val) returns () func) returns () = external;
+``` 
+
+When such a function is called in a context where an `isolated` call is expected, the argument passed for a parameter that is annotated with `@isolatedParam` should also be an `isolated` function.
+
+```ballerina
+int total = 0;
+
+function sum(int i) {
+    total += i;
+}
+
+isolated function validate(int i) {
+    if i > 100 {
+        panic error(string `Invalid Value: ${i}`);
+    }
+}
+
+function arraySum() {
+    int[] x = [1, 2, 3];
+
+    // Since `arraySum` is not `isolated`, non-isolated 
+    // function `sum` can be passed as the argument.
+    x.forEach(sum);
+}
+
+isolated function validateArray() {
+    int[] x = [1, 2, 3];
+
+    // Since `validateArray` is `isolated`, the argument 
+    // should also be an `isolated` function.
+    x.forEach(validate);
+}
+```
+
+#### Runtime
+
+The Ballerina runtime Java API package name is changed from `org.ballerinalang.jvm` to `io.ballerina.runtime`. A new `io.ballerina.api.TypeCreator` API is introduced for creating Ballerina types at runtime.
+
+#### Standard Library
+
+##### HTTP client remote function API improvement with data-binding support
+
+Response payload binding functionality is provided for an HTTP client by returning the payload according to the `targetType` defaultable parameter that is mentioned in the remote function.
+
+**Previous syntax:**
+```ballerina
+http:Response|error res = client->post("/path", req);
+```
+
+**New syntax:**
+```ballerina
+http:Response|http:Payload|error res = client->post("/path", req, json);
+// or
+json payload = <json> check client->post("/path", req, json);
```

Comment:
Since `client` is a keyword shouldn't we use some other variable name?

Reply:
@chamil321

Reply:
Thanks, I addressed the comment.